### PR TITLE
fix(math-rendering): moved the wrap and unwrap mathfunctions to the math-rendering package

### PR DESCRIPTION
**Pull request by lakatosandrei**

### CODE REVIEW

**Comment by lakatosandrei** [comment]
@edeustace 
Added

File: packages/math-rendering/src/__tests__/render-math.test.js [hunk 1]
```diff
@@ -50,10 +52,25 @@ describe.only('render-math', () => {
...
+
     renderMath([divOne, divTwo]);
+
     expect(MathJax.document).toHaveBeenCalledTimes(1);
     expect(MathJax.findMath).toHaveBeenCalledWith({
       elements: [divOne, divTwo]
     });
   });
+
+  it('wraps the math containing element the right way', () => {
```
Comment (edeustace) [inline packages/math-rendering/src/__tests__/render-math.test.js]: I would like to see more of the scenarios covered by tests.. there were some old tests in editable html that should probably be brought across: https://github.com/pie-framework/pie-lib/blob/develop/packages/editable-html/src/plugins/math/__tests__/index.test.jsx#L48-L114

File: packages/math-rendering/src/normalization.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+const MathPlugin = {};
```
Comment (edeustace) [inline packages/math-rendering/src/normalization.js]: I think `MathPlugin` is no longer the right name. maybe `BracketTypes`?

File: packages/math-rendering/src/__tests__/render-math.test.js (hunk 1 above)
Comment (lakatosandrei) [reply packages/math-rendering/src/__tests__/render-math.test.js]: I thought that I'm going to move those tests after the other PR is merged, will be moving them now.

**Pull request merged**